FirebaseVertexAI/Sources/VertexAI.swift

```diff
@@ -25,35 +25,27 @@ public class VertexAI: NSObject {
 
   /// The default `VertexAI` instance.
   ///
-  ///  - Parameter location: The region identifier, e.g., `us-central1`; see
-  ///     [Vertex AI
-  ///     regions](https://cloud.google.com/vertex-ai/docs/general/locations#vertex-ai-regions)
-  ///     for a list of supported regions.
   /// - Returns: An instance of `VertexAI`, configured with the default `FirebaseApp`.
-  public static func vertexAI(location: String) -> VertexAI {
+  public static func vertexAI() -> VertexAI {
     guard let app = FirebaseApp.app() else {
       fatalError("No instance of the default Firebase app was found.")
     }
 
-    return vertexAI(app: app, location: location)
+    return vertexAI(app: app)
   }
 
   /// Creates an instance of `VertexAI` configured with a custom `FirebaseApp`.
   ///
   ///  - Parameters:
   ///   - app: The custom `FirebaseApp` used for initialization.
-  ///   - location: The region identifier, e.g., `us-central1`; see
-  ///     [Vertex AI
-  ///     regions](https://cloud.google.com/vertex-ai/docs/general/locations#vertex-ai-regions)
-  ///     for a list of supported regions.
   /// - Returns: A `VertexAI` instance, configured with the custom `FirebaseApp`.
-  public static func vertexAI(app: FirebaseApp, location: String) -> VertexAI {
+  public static func vertexAI(app: FirebaseApp) -> VertexAI {
     guard let provider = ComponentType<VertexAIProvider>.instance(for: VertexAIProvider.self,
                                                                   in: app.container) else {
       fatalError("No \(VertexAIProvider.self) instance found for Firebase app: \(app.name)")
     }
 
-    return provider.vertexAI(location)
+    return provider.vertexAI("us-central1")
```

Comment:
Did we agree on "us-central1" or "global"?

Reply:
"us-central1" for now and will decide on whether to default to "us-central1" or "global" (given backend support) when we reintroduce the parameter.